parking_permits/admin_resolvers.py

```diff
@@ -120,22 +121,25 @@ def update_or_create_customer(customer_info):
         "driver_license_checked": customer_info["driver_license_checked"],
     }
 
-    address_info = customer_info.get("address")
+    address_info = customer_info.get("primary_address")
     if address_info:
-        kmo_address_detail = get_address_detail_from_kmo(
-            address_info["street_name"], address_info["street_number"]
-        )
-        zone = ParkingZone.objects.get_for_location(kmo_address_detail["location"])
-        address = Address.objects.create(
-            street_name=address_info["street_name"],
-            street_name_sv=kmo_address_detail["street_name_sv"],
-            street_number=address_info["street_number"],
-            city=address_info["city"],
-            city_sv=kmo_address_detail["city_sv"],
-            location=kmo_address_detail["location"],
-            zone=zone,
-            primary=True,
-        )
+        location = Point(*address_info["location"], srid=settings.SRID)
+        zone = ParkingZone.objects.get_for_location(location)
+        address = Address.objects.update_or_create(
```

Comment:
Minor but I think
`address, created = Address.objects.update_or_create(` could make sense the using `[0]`

Reply:
I think flake8 will complain if we have an unused `created` there?

Reply:
Since you are not using the created we can replace that with `address, created` -> `address, _` and flake8 will not complain about that. I think we have used that in quite a few places.

Reply:
Yeah, normally would do that too, but `_` is reserved as an alias for translation function in this module. We could also remove the alias and change all its usage, but I'd prefer simpler changes 😉

Reply:
Okay